lib/Doctrine/DBAL/Migrations/Tools/Console/Command/VersionCommand.php

```diff
@@ -55,9 +55,11 @@ protected function configure()
             ->setName('migrations:version')
             ->setDescription('Manually add and delete migration versions from the version table.')
             ->addArgument('version', InputArgument::OPTIONAL, 'The version to add or delete.', null)
+            ->addArgument('version_end', InputArgument::OPTIONAL, 'The version to add or delete.', null)
```

Comment:
the version_end is only used in combination with range right ?
If so it would be better just after the range option IMO.
Also the comment is not right on that one.